shapes/neurosciencegraph/datashapes/atlas/imagestack/schema.json

```diff
@@ -87,29 +86,35 @@
     {
       "@id": "this:SliceResolutionShape",
       "@type": "sh:NodeShape",
-      "and": [
+      "property": [
         {
-          "node": "https://neuroshapes.org/commons/quantitativevalue/shapes/QuantitativeValueShape"
+          "path": "nsg:widthResolution",
+          "name": "Width resolution",
+          "description": "Slice resolution in width",
+          "datatype": "xsd:float",
+          "minCount": 1,
+          "maxCount": 1
         },
         {
-          "property": [
+          "path": "nsg:heightResolution",
+          "name": "Height resolution",
+          "description": "Slice height",
+          "datatype": "xsd:float",
+          "minCount": 1,
+          "maxCount": 1
+        },{
+          "path": "schema:unitCode",
```

Comment:
Please use:

```
{
          "@id":"https://neuroshapes.org/commons/unit/shapes/WithUnitCodeShape"
 }
```
https://neuroshapes.org/commons/unit/shapes/WithUnitCodeShape is defined [here](https://github.com/INCF/neuroshapes/blob/master/shapes/neurosciencegraph/commons/unit/schema.json#L35).
instead of a local property path.

Reply:
I have removed this property shape entirely, since unit information will be provided with the widthResolution and height Resolution properties directly.